<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Usuario</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="0" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background : rgb(250,238,224)</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>570</width>
        <height>480</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="0" colspan="2">
        <widget class="QgsPasswordLineEdit" name="mleContrUno">
         <property name="minimumSize">
          <size>
           <width>257</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>8000</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background : rgb(255,255,255)</string>
         </property>
         <property name="maxLength">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QPushButton" name="btnOk">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255)
}
QPushButton::disabled {
background : rgb(187, 129, 13);
color : rgb(245,245,245);
border: 1px solid #adb2b5;
}</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="cbActivado">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox::indicator:unchecked {
image : url(:/plugins/AdminUsers/icons/uncheck.png);
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:checked {
    image: url(:/plugins/AdminUsers/icons/check.png);

    width: 25px;
    height: 25px;
}

QCheckBox::indicator:unchecked:disabled {
image : url(:/plugins/AdminUsers/icons/uncheck_disable.png);
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:checked:disabled{
    image: url(:/plugins/AdminUsers/icons/check_disabled.png);

    width: 25px;
    height: 25px;
}



QCheckBox {
    spacing: 4px;
    outline: none;
    padding-top: 4px;
    padding-bottom: 4px;
}

QCheckBox:focus {
    border: none;
}

QCheckBox QWidget:disabled {
    background-color: #19232D;
    color: #787878;
}

QCheckBox::indicator {
    margin-left: 4px;
    width: 16px;
    height: 16px;
}</string>
         </property>
         <property name="text">
          <string>Activado</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLabel" name="lbContrDos">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Confirmar Contraseña:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lbContrUno">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Contraseña:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Correo:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="leCorreo">
         <property name="minimumSize">
          <size>
           <width>257</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background : rgb(255,255,255)</string>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QgsPasswordLineEdit" name="mleContrDos">
         <property name="minimumSize">
          <size>
           <width>257</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>8000</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background : rgb(255,255,255)</string>
         </property>
         <property name="maxLength">
          <number>60</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Usuario:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btnAgregaRoles">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255)
}
QPushButton::disabled {
background : rgb(187, 129, 13);
color : rgb(245,245,245);
border: 1px solid #adb2b5;
}</string>
         </property>
         <property name="text">
          <string>Agregar Roles</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Apellido (s):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Nombre (s):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="leApellidos">
         <property name="minimumSize">
          <size>
           <width>257</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>257</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background : rgb(255,255,255)</string>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Roles:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="4">
        <widget class="QTableWidget" name="twRoles">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>250</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget {background-color: white;}
QHeaderView::section
{
background-color: rgb(238,225,200);
color : rgb(52, 42, 24 );
text-align: right;
font-size:12px;
height:24px
}
QHeaderView {background-color: white;}</string>
         </property>
         <column>
          <property name="text">
           <string>Rol</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Operaciones</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="leUsuario">
         <property name="minimumSize">
          <size>
           <width>257</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background : rgb(255,255,255)</string>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="leNombres">
         <property name="minimumSize">
          <size>
           <width>257</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>8000</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background : rgb(255,255,255)</string>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="btnCancelar">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background : rgb(174, 116, 0);
color : rgb(255, 255, 255)
}
QPushButton::disabled {
background : rgb(187, 129, 13);
color : rgb(245,245,245);
border: 1px solid #adb2b5;
}</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leNombres</tabstop>
  <tabstop>leApellidos</tabstop>
  <tabstop>cbActivado</tabstop>
  <tabstop>leUsuario</tabstop>
  <tabstop>mleContrUno</tabstop>
  <tabstop>mleContrDos</tabstop>
  <tabstop>leCorreo</tabstop>
  <tabstop>btnAgregaRoles</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>twRoles</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
